packages/core/src/components/essentials/tooltip-bar.ts

```diff
@@ -0,0 +1,189 @@
+import * as Configuration from "../../configuration";
+import { Tooltip } from "./tooltip";
+import { Tools } from "../../tools";
+import { DOMUtils } from "../../services";
+import { TooltipPosition, TooltipTypes } from "./../../interfaces/enums";
+
+// Carbon position service
+import Position, { PLACEMENTS } from "@carbon/utils-position";
+
+// D3 Imports
+import { mouse, select } from "d3-selection";
+
+export class TooltipBar extends Tooltip {
+
+	init() {
+		// Grab the tooltip element
+		const holder = select(this.services.domUtils.getHolder());
+		this.tooltip = DOMUtils.appendOrSelect(holder, "div.tooltip.chart-tooltip.cc-tooltip");
+
+		// Apply html content to the tooltip
+		const tooltipTextContainer = DOMUtils.appendOrSelect(this.tooltip, "div.content-box");
+
+		// listen to show-tooltip Custom Events to render the tooltip
+		this.services.events.getDocumentFragment().addEventListener("show-tooltip", e => {
+			// check the type of tooltip and that it is enabled
+			if ((e.detail.type === TooltipTypes.DATAPOINT && Tools.getProperty(this.model.getOptions(), "tooltip", "datapoint", "enabled"))
+				|| (e.detail.type === TooltipTypes.GRIDLINE && Tools.getProperty(this.model.getOptions(), "tooltip", "gridline", "enabled")) ) {
+
+				const hoveredElement = e.detail.hoveredElement.node();
+
+				// if there is a provided tooltip HTML function
+				if (Tools.getProperty(this.model.getOptions(), "tooltip", "customHTML")) {
+					tooltipTextContainer.html(this.model.getOptions().tooltip.customHTML(hoveredElement));
+				} else {
+					if (e.detail.multidata) {
+						// multi tooltip
+						tooltipTextContainer.html(this.getMultiTooltipHTML(e.detail.multidata));
```

Comment:
`getMultiTooltipHTML` Maybe `getMultilineTooltipHTML`?

Reply:
+1 for `getMultilineTooltipHTML`

Reply:
this should be fixed now